awpy/types.py

```diff
@@ -482,6 +482,8 @@ class GameRound(TypedDict):
     weaponFires: Optional[list[WeaponFireAction]]
     flashes: Optional[list[FlashAction]]
     frames: Optional[list[GameFrame]]
+    mvp: Players
```

Comment:
I dont think the `gameRounds` dict contains a `Players` dict for the key `mvp` from your implementation.

I think it should have an `MVPSteamID: Optional[int]` and the same for MVPName as optional string.